compose/install/index.md

```diff
@@ -3,30 +3,73 @@ description: How to install Docker Compose
 keywords: compose, orchestration, install, installation, docker, documentation
 title: Install Docker Compose
 toc_max: 2
+redirect_from:
+- /compose/compose-desktop/
 ---
 
-On this page you can find a summary of the available options for installing Docker Compose. 
+On this page you can find information on how to get and install Compose.
 
-## Compose prerequisites
+## Install Compose
 
-* Docker Compose requires Docker Engine.
-* Docker Compose plugin requires Docker CLI.
+**To get Compose:**
 
-## Compose installation scenarios
-You can run Compose on macOS, Windows, and 64-bit Linux. Check what installation scenario fits your needs. 
+* If you have Docker Desktop, you've got a full Docker installation, including Compose.
 
-Are you looking to:
+* If you don't have Docker yet installed, or just need to install Compose, check the options below:
```

Comment:
```suggestion
```

I think the "If you don't have Docker yet installed" should be included with the prior sentence about Docker Desktop. So that way, we address DD entirely with the first point:

> If you've previously installed Docker Desktop, then you already have a full Docker installation that includes Compose. (If not, download links for Docker Desktop by OS are in the table below.)



Then, that leaves the statement: 
> If you…just need to install Compose, check the options below:

This could be more helpful. Under what circumstances would the user need to install Compose outside Docker Desktop? And how do they tell the difference between whether they should install it as a plugin, or standalone?

(I address this point a bit in the "Table Description" feedback below)

Reply:
See closing comment.